moped-database/views/project_list_view.sql

```diff
@@ -1,3 +1,4 @@
+-- latest version 1678894870998_remove_cols_from_geo_views
```

Comment:
Nice touch! I've struggled to find the latest migration affecting this view more than once!